iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/NetworkXml.java

```diff
@@ -216,6 +216,7 @@ private static void writeExtensions(Network n, NetworkXmlWriterContext context,
 
             Collection<? extends Extension<? extends Identifiable<?>>> extensions = identifiable.getExtensions().stream()
                     .filter(e -> getExtensionXmlSerializer(options, e) != null)
+                    .filter(e -> context.getOptions().withExtension(e.getName()))
```

Comment:
This test is already performed in getExtensionXmlSerializer
```
    private static ExtensionXmlSerializer getExtensionXmlSerializer(ExportOptions options, Extension<? extends Identifiable<?>> extension) {
        if (options.withExtension(extension.getName())) {
        ....
        }
     }
```

Reply:
You're right!